internal/setup/setup.go

```diff
@@ -670,9 +720,15 @@ func Select(release *Release, slices []SliceKey) (*Selection, error) {
 					}
 					return nil, fmt.Errorf("slices %s and %s conflict on %s", old, new, newPath)
 				}
-				continue
+			} else {
+				paths[newPath] = new
+			}
+			switch newInfo.Generate {
```

Comment:
Can you add a comment here saying that an unknown generate should only be an error at runtime if the slice is selected?

Reply:
Done: ad54c29.